### PR TITLE
Add darwinia test network

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
There can only be one native currency - wonder why you need 2

**Comment by AsceticBear** [comment]
> There can only be one native currency - wonder why you need 2

Let me explain the details more. Darwinia network is a decentralized cross-chain bridge based on substrate framework. Due to the nature of substrate, we support two currencies now, RING, KTON.

The Pangolin network is a test network built by Darwinia to support ethereum evm which allows users to migrate existing Dapps to the Darwinia network without much modification.

**Comment by ligi** [comment]
thanks for the info! But when a (EVM) smart-contract is executed - from which of the currencies does it take the gas?

**Comment by AsceticBear** [comment]
> thanks for the info! But when a (EVM) smart-contract is executed - from which of the currencies does it take the gas?

The first one, RING.

**Comment by ligi** [comment]
please set RING as native currency then. We can also add a new field maybe for a secondary currency. But of that I would need more information how this information can be used then.

**Comment by AsceticBear** [comment]
> please set RING as native currency then. We can also add a new field maybe for a secondary currency. But of that I would need more information how this information can be used then.

Done, Please check again later.

File: _data/chains/eip155-43.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Darwinia Pangolin Testnet",
+  "chain": "pangolin",
+  "network": "free testnet",
+  "rpc": [],
+  "faucets": [],
+  "nativeCurrency1": {
```
Comment (ligi) [inline _data/chains/eip155-43.json:7]:
```suggestion
  "nativeCurrency": {
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by ligi** [comment]
I am happy with it now - but CI is missing the shortName field

**Pull request merged**